pkg/grpc/grpc_util.go

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Will it be ok to rename the file to `grpc.go`?
Will it be ok to have this under `pkg/grpc/v1alpha1/`